modal-go/function_call.go

```diff
@@ -0,0 +1,44 @@
+package modal
+
+import (
+	"context"
+	"fmt"
+
+	pb "github.com/modal-labs/libmodal/modal-go/proto/modal_proto"
+)
+
+// FunctionCall references a Modal Function Call. Function Calls are
+// Function invocations with a given input. They can be consumed
+// asynchronously (see Get()) or cancelled (see Cancel()).
+type FunctionCall struct {
+	FunctionCallId string
+	ctx            context.Context
+}
+
+// Gets the ouptut for a FunctionCall
```

Comment:
Could we correct typos in the comments and start the comment with the name of the method?

```go
// Get waits for the output of a FunctionCall.
```

Also, doc comments should be complete sentences. Please see https://tip.golang.org/doc/comment here for the guidelines!